service/src/test/java/bio/terra/catalog/api/DatasetApiControllerTest.java

```diff
@@ -74,6 +74,18 @@ void listDatasets() throws Exception {
         .andExpect(jsonPath("$.result[0].id").value("id"));
   }
 
+  @Test
+  void EmptyListDatasets() throws Exception {
+    DatasetsListResponse response = new DatasetsListResponse();
+    when(datasetService.listDatasets(user)).thenReturn(response);
+    mockMvc
+        .perform(get(API))
+        .andExpect(status().isOk())
+        .andExpect(header().string("Cache-Control", "no-store"))
+        .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+        .andExpect(jsonPath("$.result").isEmpty());
```

Comment:
This check will produce a false positive. That is, even if you back out the yaml change, this test will continue to pass. I think that's because `isEmpty()` matches both missing and empty arrays in the JSON response. Using this assert instead will correctly test the change. You can verify this by reverting the yaml change locally and running the test.
```suggestion
        .andExpect(jsonPath("$.result").isArray());
```